lib/src/main/res/xml/network_security_config.xml

```diff
@@ -0,0 +1,7 @@
+<?xml version="1.0" encoding="utf-8"?>
+<network-security-config>
+    <!-- Allow local cleartext networking with PDC Server -->
+    <domain-config cleartextTrafficPermitted="true">
+        <domain includeSubdomains="false">127.0.0.1</domain>
```

Comment:
Can we / show we limit it to port `13276`?

Reply:
Couldn't find anything related with ports under the spec: https://developer.android.com/training/articles/security-config

Reply:
:+1: